src/metrics/site_level.jl

```diff
@@ -14,7 +14,7 @@ Get metric results applied to the location-level at indicated time (or across ti
 # Returns
 Named Vector of \$N\$ elements, where \$N\$ is the number of sites.
 """
-function per_loc(metric, data::NamedDimsArray{D,T,3,A})::NamedDimsArray where {D,T,A}
+function per_loc(metric, data::NamedDimsArray{D,T,N,A})::NamedDimsArray where {D,T,A}
```

Comment:
Ah, could you remove `where {D,T,A}` here please? It doesn't do anything here as I'm not specifying any types.

Reply:
Ah, no turns out you do need it, so add `N` to the list:

`where {D,T,N,A}`

Reply:
Don't forget to update docstrings as well